python/cudf_polars/cudf_polars/experimental/dask_registers.py

```diff
@@ -128,3 +129,20 @@ def _(header: ColumnHeader, frames: tuple[memoryview, memoryview]) -> Column:
             # Copy the second frame (the gpudata in host memory) back to the gpu
             frames = frames[0], plc.gpumemoryview(rmm.DeviceBuffer.to_device(frames[1]))
             return Column.deserialize(header, frames)
+
+    @sizeof_dispatch.register(Column)
+    def _(x: Column) -> int:
+        """The total size of the device buffers used by the Column."""
+        ret = 0
+        if x.obj.data() is not None:
+            ret += x.obj.data().nbytes
+        if x.obj.null_mask() is not None:
+            ret += x.obj.null_mask().nbytes
+        if x.obj.children() is not None:
+            ret += sum(sizeof_dispatch(c) for c in x.obj.children())
+        return ret
```

Comment:
Can we move these definitions into cached properties on the `Column` and `Table` objects in pylibcudf. Then the registration can trivially do `def _(x: Column) -> int: return x.nbytes`

Rationale: we might want to have spilling enabled for the synchronous (non-dask) task execution as well.

Reply:
Good point!
Calling it `.device_buffer_size` (or do you prefer `.nbytes`?).
Or maybe `.device_alloc_size` is better?